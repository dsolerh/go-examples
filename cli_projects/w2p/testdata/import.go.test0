package example

// nothing to replace
import (
	one "github.com/package/do/not/replace/this/one"
	two "github.com/package/do/not/replace/two"
	"github.com/package/do/not/replace/three"
)

func example() {
    one.Print()
    two.Print()
    three.Print()
}