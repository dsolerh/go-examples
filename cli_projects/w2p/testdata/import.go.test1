package example

// replace: do/not/exist -> exist
import (
	one "github.com/package/do/not/replace/this/one"
	two "github.com/package/do/not/exist/two"
	"github.com/package/do/not/exist/three"
)

func example() {
    one.Print()
    two.Print()
    three.Print()
}