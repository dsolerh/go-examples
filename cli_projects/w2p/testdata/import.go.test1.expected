package example

// replace: do/not/exist -> exist
import (
	one "github.com/package/do/not/replace/this/one"
	two "github.com/package/exist/two"
	"github.com/package/exist/three"
)

func example() {
    one.Print()
    two.Print()
    three.Print()
}